foundry/config.d.ts

```diff
@@ -1454,7 +1454,9 @@ declare const CONFIG: {
      */
     speakingHistoryLength: number;
   };
-} & Record<string, unknown>;
```

Comment:
I wouldn’t remove this so that it’s still possible for people who don’t want to bother with it too much to just put additional stuff into the config.

Reply:
Its still there, just that its an extends instead. You can't really do the & thing with a namespace.

Reply:
Ah, didn't see it, sorry. I think it's fine then.

Regarding `unknown` screaming if you want to access anything that's nested and that we should use `any` instead: I don't agree. It's the right thing to cream because the type system cannot guarantee anything for you. So you basically need to tell it "Shut up, I know what I'm doing" by casting it to the right thing. If you use `any`, things can fail silently without the developer even noticing that anything dangerous might be happening.

I know that using `any` would be a bit more convenient sometimes. It is ultimately a trade off between convenience and type safety and a discussion we had often already, not with respect to this specific occurrence but it happens in other places, too, e.g. the return type of `getFlag` etc. It's not something we need to discuss / resolve here.

Reply:
Of course, I just mean that I don't think unknown solves the problem of people who don't want to bother with it to much. It forces you to bother so if that was the goal I'm not sure it accomplishes it. But it is definitely more type safe and its why it exists.